main.go

```diff
@@ -199,7 +264,11 @@ func ProcessAPI(shortName string, api *openapi3.Swagger) *OpenAPI {
 			optionalParams := getOptionalParams(params)
 			short := operation.Summary
 			if short == "" {
-				short = name
```

Comment:
Here is a slight change to the current logic, where if no `summary` property is provided we attempt use the `description` in case it's available before falling back to the operation name.

Reply:
We tend to put long Markdown descriptions in this field, so I would rather see it use the operation ID than the description. Thoughts?

Reply:
Makes sense, we should probably fix our OAS spec to specify a `summary` instead. Reverted.